decidim-admin/app/assets/stylesheets/decidim/admin/modules/_action-icon.scss

```diff
@@ -11,3 +11,17 @@
 .action-icon.action-icon--remove{
   color: $alert-color;
 }
+
+.action-icon--hide{
+  position: relative;
+
+  &:before{
+    border-left: 1px solid;
+    bottom: 0;
+    content: "";
+    left: 36%;
+    position: absolute;
+    top: 0;
+    transform: rotate(-45deg);
+  }
+}
```

Comment:
I moved the stuff to decidim-design-admin and improved [here](https://github.com/decidim/design-admin/pull/17).

So if I understood how you work with this stuff correctly, once this is added to the design repo, the design is "vendored" here (meaning the files are copied from that repo and should never be changed)?